incubator/eventrouter/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Any reason why you changed the default implementation for `name` and `fullname`?